_posts/2025-02-06-highlights-2024.md

```diff
@@ -64,6 +64,8 @@ Further reading:
 
 ### Experimental WebAssembly backend in Scala.js
 
+*By the Scala Open Source team at VirtusLab*
```

Comment:
I think Rikito wrote some of this, but if I am mistaken let me know.

Reply:
I wonder if there might be any confusion about whether "by" refers to who wrote the newsletter section, or who did the engineering work being described?

Reply:
I didn't want to add a whole new formula, just similar in case of jetbrains, which I think is about who wrote that section? We added the sections for Scala CLI and Metals, so figured it's enough to mention it. Wasn't sure about WASM though.

I can also change it to `Work mostly done by VirtusLab Open Source Team`, but that seems too on the nose.